pages/common/gcpdiag.md

```diff
@@ -0,0 +1,25 @@
+# gcpdiag
+
+> Google Cloud Platform troubleshooting and diagnostics tool.
+> Run in a docker container or in GCP Cloudshell.
+> More information: <https://github.com/GoogleCloudPlatform/gcpdiag>.
+
+- Run `gcpdiag` on your project, returning all rules:
+
+`gcpdiag lint --project={{gcp_project_id}}`
```

Comment:
```suggestion
- Run `gcpdiag` on specific project, returning all rules:

`gcpdiag lint --project="{{id}}"`
```

Optional fix: `specific` may sound better.
Optional fix: placeholder can be simplified.
Optional fix: placeholder can be quoted to protect from misinterpretation by POSIX-compatible shell.